apps/prairielearn/src/ee/pages/instructorAiGenerateJobDetails/instructorAiGenerateJobDetails.html.ts

```diff
@@ -0,0 +1,79 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { Job } from '../../../lib/db-types.js';
+
+export function AiGenerateJobDetailsPage({
+  resLocals,
+  job,
+}: {
+  resLocals: Record<string, any>;
+  job: Job;
+}) {
+  return html` <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals })}
+      </head>
+      <body hx-ext="loading-states">
+        ${renderEjs(import.meta.url, "<%- include('../../../pages/partials/navbar'); %>", {
+          navPage: 'course_admin',
+          ...resLocals,
+        })}
+        <main id="content" class="container-fluid">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Generation Job Results</div>
+            <div class="card-body">
+              <div class="mr-auto">
+                <span class="card-title"> Prompt </span>
+              </div>
+              <div id="card-prompt">
+                <pre id="output-prompt" class="bg-dark text-white rounded p-3">
+${job.data['prompt']} 
+              </pre
+                >
+              </div>
+
+              <div class="mr-auto">
+                <span class="card-title"> Generated HTML </span>
+              </div>
+              <div id="card-html">
+                <pre id="output-html" class="bg-dark text-white rounded p-3">
+${job.data['html']} 
+              </pre
+                >
```

Comment:
The `<pre>` tag is whitespace-sensitive, so you'll want to make sure that the opening tag, content, and closing tag all appear on the same line (plus or minus some Prettier formatting):

```suggestion
                <pre id="output-html" class="bg-dark text-white rounded p-3">
${job.data['html']} </pre>
```

This applies to all other instances in this file.

Reply:
changed in 3c8bafd1470728a8c79b488213de1dfae0aed79e